### PR TITLE
enable AWS Backup

### PR DESCRIPTION
**Pull request by michaelwittig**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by michaelwittig** [comment]
test

File: module.yml [hunk 1]
```diff
@@ -269,11 +279,63 @@ Resources:
       - 'Fn::ImportValue': !Sub '${AlertingModule}-Arn'
       OKActions:
       - 'Fn::ImportValue': !Sub '${AlertingModule}-Arn'
+  BackupVault: # cannot be deleted with data
```
Comment (andreaswittig) [inline module.yml]: What is the difference between AWS Backup and the property `PointInTimeRecoverySpecification` of the `AWS::DynamoDB::Table` resource? Two separate things? If so, should we also enable `PointInTimeRecoverySpecification`?

Comment (andreaswittig) [reply module.yml]: I assume AWS Backup is creating DynamoDB on-demand backups. So we should enable point-in-time recovery as well?

Comment (michaelwittig) [reply module.yml]: good point. done

**Comment by michaelwittig** [comment]
test

**Pull request merged**